docs/using/syntax.md

```diff
@@ -416,10 +414,41 @@ you may start the content directly after the directive name.
 ```{note} Notes require **no** arguments, so content can start here.
 ```
 
+For special cases, MySt also offers the `eval-rst` directive.
+This will parse the content **as ReStructuredText**:
+
+````md
+```{eval-rst}
+.. figure:: img/fun-fish.png
+  :width: 100px
+  :name: rst-fun-fish
+
+  Party time!
+
+A reference from inside: ref:`rst-fun-fish`
```

Comment:
Typo here

Reply:
Ah damn thanks, so close to perfection 😆